eugl/gqa/tasks.py

```diff
@@ -55,17 +71,455 @@
 # rather than find and read the internal cfg.
 # Similar to how wagl and tesp do it
 CONFIG = luigi.configuration.get_config()
-CONFIG.add_config_path(resource_filename('gqa', 'gqa-defaults.luigi.cfg'))
+
+# TODO purge this
+# CONFIG.add_config_path(resource_filename('gqa', 'gqa-defaults.luigi.cfg'))
 
 # TODO variable change; scene id is landsat specific, level1 is more generic
 # TODO remove refs to l1t (landsat specific)
 
 
+class GQATask(luigi.Task):
```

Comment:
Can we add a comment that this task is only useful with the singlefile workflow in wagl; (determined by the DataStandardisation dependency).

Reply:
Done.